### PR TITLE
Add 'Reason' to the request expandable content

### PR DESCRIPTION
**Pull request by lgalis**
Add 'Reason' to the request expandable content.
Update package versions.

Screenshot from 2019-06-19 11-10-02

### CODE REVIEW

**Comment by lgalis** [comment]
@Hyperkid123 - please review

File: src/smart-components/request/expandable-content.js [hunk 1]
```diff
@@ -4,7 +4,15 @@ import { Link } from 'react-router-dom';
 import { TextContent, Text, TextVariants, Level, LevelItem, Button } from '@patternfly/react-core';
 import { isRequestStateActive } from '../../helpers/shared/helpers';
 
-const ExpandableContent = ({ id, content, state }) => {
+const expandedItem = (title, detail) => {
```
Comment (Hyperkid123) [inline src/smart-components/request/expandable-content.js]: I think we be using it as a component not as a pure function. Its a bit confisung

```jsx
const ExpandedItem = ({title, detail}) => ...
```

File: src/smart-components/request/expandable-content.js [hunk 2]
```diff
@@ -31,21 +39,9 @@ const ExpandableContent = ({ id, content, state }) => {
...
-            <Text className="data-table-detail heading" component={ TextVariants.small }>Platform</Text>
-            <Text className="data-table-detail content"
-              component={ TextVariants.h5 }>{ content ? content.platform : 'Unknown' }</Text>
-          </TextContent>
-          <TextContent>
-            <Text className="data-table-detail heading" component={ TextVariants.small }>Comments</Text>
-            <Text className="data-table-detail content"
-              component={ TextVariants.h5 }>{ content ? content.comments : 'Unknown' }</Text>
-          </TextContent>
+          { expandedItem('Portfolio', content ? content.portfolio : 'Unknown') }e
```
Comment (Hyperkid123) [inline src/smart-components/request/expandable-content.js]: ^^
```jsx
<ExpandedItem ... />
```

And i believe there is a trailing character `e` after closing curly bracket

File: src/smart-components/request/expandable-content.js [hunk 3]
```diff
@@ -57,7 +53,8 @@ ExpandableContent.propTypes = {
   id: PropTypes.string,
   content: PropTypes.obuidt,
   uname: PropTypes.string,
-  state: PropTypes.string
+  state: PropTypes.string,
+  reason: PropTypes.string
```
Comment (Hyperkid123) [inline src/smart-components/request/expandable-content.js]: Reason looks like to be always required.

**Pull request merged**